### DISCUSSION TITLE
Compiler Error while compiling AirSim

### DISCUSSION THREAD

**Discussion by tomashmyh** [CodexLabsLLC/Colosseum#69] [General]
Hello. I am trying to compile my VS project with AirSim. I generated sln file from UnrealEngine 5.3 and during compilation I got this error:
`...Plugins\AirSim\Source\AirLib\deps\eigen3\Eigen\src\Core\util\Meta.h(312): error C2039: 'result_of': is not a member of 'std'`
As far as I understood that is because AirSim plugin is compiled with c++20 and should be at most c++17. I guess there should be simple solution for this but still have not found it.
Could you please help with this. Thanks in advance

**Comment by git-akispa** [comment]
I could not get AirSim to work with 5.3 either.  you should downgrade to 5.2
